src/arrayMethodSort.js

```diff
@@ -5,8 +5,30 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    const compare = compareFunction || toSort;
```

Comment:
instead of this check, just add the default value for comparateFunction argument